bindings/python/pymongoarrow/schema.py

```diff
@@ -62,10 +63,24 @@ def _normalize_mapping(mapping):
 
     def _get_projection(self):
         projection = {"_id": False}
-        for fname, _ in self.typemap.items():
-            projection[fname] = True
+        for fname, ftype in self.typemap.items():
+            if isinstance(ftype, ListType):
+                projection[fname] = self._get_field_projection_value(ftype.value_field.type)
+            else:
+                projection[fname] = self._get_field_projection_value(ftype)
```

Comment:
I think we can collapse this if/else into just a single line:
```
```

Reply:
Good catch, since lists are projected identically to whatever they contain.